tests/mbt/model/ccv.qnt

```diff
@@ -93,6 +99,29 @@ module ccv_types {
             // a monotonic strictly increasing and positive ID that is used
             // to uniquely identify the Vscs sent to the consumer chains.
             runningVscId: int,
+
+            // For every consumer chain, stores the consumer address assigned by each validator.
+            validatorConsumerPubKey: Chain -> (Node -> ConsumerAddr),
+
+            // the history of validator sets on the provider, but with the key assignments applied.
+            // This is needed to check invariants about the validator set when key assignments are in play.
+            keyAssignedValSetHistory: Chain -> VotingPowerHistory,
+
+            // For every consumer chain, holds the provider validator for each assigned consumer key.
+            validatorByConsumerAddr: Chain -> (ConsumerAddr -> Node),
```

Comment:
This seems to be the reverse of `validatorConsumerPubKey` so it might make sense to put under `validatorConsumerPubKey`. Also, this is called `ConsumerAddr` versus `ConsumerPubKey` above. Might make sense to have them consistent (I personally prefer the name with the `Addr` because `PubKey` feels to implementation specific).